### PR TITLE
Closes #123 refresh data

### LINKED ISSUE #123: General Issue: Refresh of data
### Background Information

It's time to refresh the data using the latest versions of the packages.

### Definition of Done

Using the following versions:
- admiral 1.3.1
- admiralonco ~1.2.0~ 1.3.0 (available on 01-SEP-2025 on CRAN)
- admiralophtha 1.3.0
- admiralvaccine **dev version**: depends on [issue #260](https://github.com/pharmaverse/admiralvaccine/issues/260), which must be completed first. ✅ **dev version 0.5.0.9000**
- admiralpeds 0.2.1: release to be completed first ([issue #108 ](https://github.com/pharmaverse/admiralpeds/issues/108)) ✅
- admiralmetabolic **dev version**: depends on [issue #129]( https://github.com/pharmaverse/admiralmetabolic/issues/129), which must be completed first. ✅ **dev version 0.2.0.9000**
- pharmaversesdtm 1.3.1

### PR DESCRIPTION
**Pull request by Fanny-Gautier**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build pharmaverseadam site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new datasets occur on the "[Reference](https://pharmaverse.github.io/pharmaverseadam/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Review by manciniedoardo — APPROVED** [review]
Just a small comment!

File: NEWS.md [hunk 1]
```diff
@@ -2,12 +2,14 @@
 
 ## New features
 
+  - Refreshed `{pharmaverseadam}` datasets using `{admiral}` 1.3.1.9006 (development version), `{admiralonco}` 1.2.0, `{admiralophtha}` 1.3.0, `{admiralvaccine}` 0.5.0.9000 (development version), `{admiralpeds}` 0.2.1, `{admiralmetabolic}` 0.2.0.9000 (development version), `{pharmaversesdtm}` 1.3.1. (#123)
+  
+    _Note: This uses development version 1.3.1.9006 of `{admiral}` (with the updated `ADLBHY` template), as well as development versions 0.5.0.9000 of `{admiralvaccine}` and 0.2.0.9000 of `{admiralmetabolic}` (with the templates directory issues fixed)._
```
[the remark this answers is not in the document]
Comment (Fanny-Gautier) [reply NEWS.md:7]: The note below is for #105 and {admiralmetabolic} only. Shall I fully delete the note for #105 ?

**Comment by shajoezhu** [comment]
reverse dependency check looks ok. https://github.com/pharmaverse/pharmaverseadam/actions/runs/17465083893

**Review by Lina2689 — APPROVED** [review]
